src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -204,29 +231,26 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
   const showSoundCloudRadio =
     (currentVenue?.showRadio && isSoundCloud) ?? false;
 
+  const showRadio = showNormalRadio || showSoundCloudRadio;
+  const volumeIcon = !volume ? faVolumeMute : faVolumeUp;
```

Comment:
Could flip the operands to remove the negation.
```suggestion
  const volumeIcon = volume ? faVolumeUp : faVolumeMute;
```

Reply:
fixed 433dfe8